src/handlers/entryModification/modifyEntryState.ts

```diff
@@ -80,6 +75,15 @@ export const modifyEntryState: RequestHandler = (req, res) => {
         });
     }
 
+    // validate permissions for featured entries
+    if (currentState === EntryStates.Featured || newState === EntryStates.Featured) {
+        if (token.user.permissionLevel < UserPermissionLevels.Owner) {
+            res.setHeader(`Perms-RequiredLevel`, UserPermissionLevels.Owner);
+            res.setHeader(`Perms-CurrentLevel`, token.user.permissionLevel);
+            return res.sendStatus(401);
```

Comment:
I assume here the user has credentials? This should be a 403 (forbidden) status if credentials are provided but aren't sufficient.
```suggestion
            return res.sendStatus(403);
```

Reply:
Yeah we'd have to change all our endpoints for that (and the API docs), maybe another PR.